pkg/fiber-otel/config.go

```diff
@@ -8,13 +8,15 @@ import (
 type Config struct {
 	Tracer                trace.Tracer
 	TracerStartAttributes []trace.SpanStartOption
-	SpanName              string
-	LocalKeyName          string
+	// SpanName is a template for span naming.
+	// The scope is fiber context.
+	SpanName     string
+	LocalKeyName string
 }
 
 // ConfigDefault is the default config
 var ConfigDefault = Config{
-	SpanName:     "http/request",
+	SpanName:     "{{ .Method }} {{ .Route.Path }}",
```

Comment:
I'm not sure if default should be "GET /some/route". Looked at otel spec and it seams to better fit "/some/route". 
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name

It would be awesome to add some unit tests. 

Other then that everything looks good for me. Super thanks for this PR.